src/controllers/cartsController.js

```diff
@@ -0,0 +1,23 @@
+const cartService = require("../models/cartsDao");
+
+const cartIn = async (req, res) => {
+  try {
+    const userId = req.userId;
+
+    const { productsId } = req.params;
```

Comment:
데이터는 body로 받아 주시고, 수량도 같이 받아 주세요.
```suggestion
    const { productId, quantity } = req.body;
```